style.css

```diff
@@ -1,296 +1,493 @@
-@media only screen and (max-width: 600px) {
-  @font-face {
-    font-family: Poppins-bold;
-    src: url(./Poppins.ttf);
-  }
+@font-face {
+  font-family: Poppins-bold;
+  src: url(./Poppins.ttf);
+}
 
-  @font-face {
-    font-family: Poppins-regular;
-    src: url(./Poppins-Regular.ttf);
-  }
+@font-face {
+  font-family: Poppins-regular;
+  src: url(./Poppins-Regular.ttf);
+}
 
-  body {
-    background-color: #f7f7f9;
-    margin: 0;
-    padding: 0;
-  }
+body {
+  background-color: #f7f7f9;
+  margin: 0;
+  padding: 0;
+}
 
-  .first-wrapper {
-    margin: 0;
-    background-image: url("./assets/bg\ image.png.png");
-    border-radius: 0 0 0 50px;
-    background-color: #fff;
-    padding: 20px;
-  }
+.first-wrapper {
+  margin: 0;
+  background-image: url("./assets/bg\ image.png.png");
+  border-radius: 0 0 0 50px;
+  background-color: #fff;
+  padding: 20px;
+}
 
-  .header {
-    background-color: #fff;
-    margin: 0;
-    padding: 0 20px;
-  }
+.header {
+  background-color: #fff;
+  margin: 0;
+  padding: 0 20px;
+}
 
-  nav {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    margin-left: 20px;
-    margin-right: 20px;
-  }
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-left: 20px;
+  margin-right: 20px;
+}
 
-  a {
-    color: #6070ff;
-    text-decoration: none;
-  }
+.nav-list {
+  display: none;
+  width: 0;
+  height: 0;
+}
 
-  h1 {
-    font-size: 40px;
-    font-weight: 700;
-    line-height: 52px;
-    padding-top: 30px;
-  }
+a {
+  color: #6070ff;
+  text-decoration: none;
+}
 
-  p {
-    font-size: 15px;
-    font-weight: 400;
-    line-height: 24px;
-  }
+h1 {
+  font-size: 40px;
```

Comment:
Hi @Olamarx ,
I have made this change.

Reply:
Hi @Olamarx ,
Here you are referring to my mobile version which is different from the desktop version. My css for the desktop version is below the css for the mobile app version.
You also referred to my coding partner @munyite001 instead of referring to me @Victoria129 . Could you kindly re-review my pull request?